README.mdx

```diff
@@ -0,0 +1,198 @@
+# TODO: Expand on outline
+
+> :warning: This guidance is in prep for a Playground tutorial, but attachments must first be enabled on Playground for
+this guidance to make sense.
+
+# KittyVerse + KittyHats
+
+To showcase how attachments unlock permissionless composability on Flow, this tutorial will walk you though making your
+own cat-inspired NFT collection and a wardrobe of hats to go along with them. Let's put some hats on some cats!
+
+## What is composability?
+
+Before digging in, let's talk about what we mean by **_composability_**.
+
+<Callout type="info">
+
+Composability is the ability for existing components in one construction to be reused and even extended for another 
+purpose. <br/>
+
+Think of composable architectures like [building with legos](https://twitter.com/cdixon/status/1450462593232089088?s=20) 
+- you build something you need as a block or set of blocks and someone else can come along and use those same blocks to 
+build something entirely new.
+
+Composability on Flow, especially with attachments is permissionless because not only can you build with existing 
+components, but no one can stop you from doing so!
```

Comment:
```suggestion
Composability on Flow, especially with attachments is permissionless because not only can you build with existing 
components, but no one can stop you from doing so! Hence the term permissionless!
```